examples/typescript/src/index.html

```diff
@@ -34,8 +34,11 @@ <h3> Program </h3>
                     <option value="460800">460800</option>
                     <option value="230400">230400</option>
                     <option value="115200">115200</option>
+                    <option value="74880">74880</option>
```

Comment:
Do we require this for flashing? Only the console baudrate has to be lower at 74880 for 26MHz chips

Reply:
Removed